primitives/src/merkle_tree/field_based_mht/parameters/tweedle_fr.rs

```diff
@@ -0,0 +1,78 @@
+use algebra::{
+    fields::tweedle::Fr as TweedleFr,
+    biginteger::BigInteger256,
+    field_new,
+};
+use crate::{
+    crh::TweedleFrPoseidonHash,
+    merkle_tree::field_based_mht::FieldBasedMerkleTreePrecomputedEmptyConstants
+};
+
+pub const TWEEDLE_MHT_POSEIDON_PARAMETERS: FieldBasedMerkleTreePrecomputedEmptyConstants<'static, TweedleFrPoseidonHash> =
+    FieldBasedMerkleTreePrecomputedEmptyConstants {
+        nodes: &[
```

Comment:
These constants need to be recomputed when using the new Poseidon parameters from [UpdatedPoseidonParameters](https://github.com/HorizenOfficial/ginger-lib/tree/UpdatedPoseidonParameters)